index.html

```diff
@@ -1,375 +1,341 @@
-              </div>
-              <div class="btnt">
-                <div class="btn-T">
-                  <button type="button">See Project</button>
+    </section>
+
+    <section class="work-section" id="portfolio">
+        <div class="container">
+            <div class="row justify-content-center">
+                <div class="col-md-12 ">
+                    <div class="row project overflow-hidden mb-4 shadow-sm position-relative">
+                        <div class="col-auto p-4 d-lg-block">
+                            <img src="./image/portfolio1.png" alt="image portfolio-1">
+                        </div>
+                        <div class="col p-4 details">
+                            <h3 class="mb-0 title">Tonic</h3>
+                            <div class="mb-1 text-muted work-info">
+                                <ul>
+                                    <li class="intro">Canopy</li>
+                                    <li><i class="fa fa-circle"></i> Back End Dev</li>
+                                    <li><i class="fa fa-circle"></i> 2015</li>
+                                </ul>
+                            </div>
+                            <p class="card-text">
+                                A daily selection of privately personalized reads; no accounts or sign-ups required.
+                            </p>
+
+                            <div class="tags">
+                                <span class="badge rounded-pill bg-tag">html</span>
+                                <span class="badge rounded-pill bg-tag">css</span>
+                                <span class="badge rounded-pill bg-tag">javascript</span>
+                            </div>
+                            <button type="button" id="btn-1" class="btn btn-outline-primary btn-lg">See project</button>
+                        </div>
+                    </div>
+
+                    <div class="row project overflow-hidden mb-4 shadow-sm position-relative">
+                        <div class="col-auto p-4 d-lg-block">
+                            <img src="./image/portfolio2.png" alt="image portfolio-1">
+                        </div>
+                        <div class="col p-4 details">
+                            <h3 class="mb-0 title">Multi-Post Stories</h3>
+                            <div class="mb-1 text-muted work-info">
+                                <ul>
+                                    <li class="intro">FACEBOOK</li>
+                                    <li> <i class="fa fa-circle"></i> Full Stack Dev</li>
+                                    <li><i class="fa fa-circle"></i> 2015</li>
+                                </ul>
+                            </div>
+                            <p class="card-text">
+                                Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+                            </p>
```

Comment:
* Kindly load each project content from the array of objects created and not hard code it using `HTML` doing so would make addition and subtraction of projects in the future easy plus it is a requirement for this project [Check Here](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=For%20each%20project,the%20page%20loads.)